pkg/stanza/flush/flush.go

```diff
@@ -6,8 +6,12 @@ package flush // import "github.com/open-telemetry/opentelemetry-collector-contr
 import (
 	"bufio"
 	"time"
+
+	"github.com/tilinna/clock"
 )
 
+var Clock = clock.Realtime()
```

Comment:
I'd feel better if we don't have to depend on any 3rd party library for the production code. We should be able to do this instead and achieve the same benefit for tests.
```suggestion
var now = time.Now
```

Then in tests just assign a new function to `now`